CovertInfiltration/Src/CovertInfiltration/Classes/X2EventListener_Infiltration.uc

```diff
@@ -204,7 +204,7 @@ static protected function EventListenerReturn ShouldEmptySlotsOnActionRemoval(Ob
 	
 	if (Action == none || Tuple == none || Tuple.Id != 'CovertAction_RemoveEntity_ShouldEmptySlots') return ELR_NoInterrupt;
 
-	if (class'X2Helper_Infiltration'.static.IsInfiltrationAction(Action))
+	if (!Action.bStarted && class'X2Helper_Infiltration'.static.IsInfiltrationAction(Action))
```

Comment:
> do not kick people from finished infiltration 

> do not kick people from **finished** infiltration 

How can it be finished if it hasn't started? 

(the check should be whether the action IS started)